app/views/services/index.njk

```diff
@@ -5,7 +5,11 @@
   Choose service - GOV.UK Pay
 {% endblock %}
 
-{% block beforeContent %}{% endblock %}
+{% block beforeContent %}
+  {% if isDegatewayed %}
+    {% include "includes/degateway-banner.njk" %}
+  {% endif %}
+{% endblock %}
```

Comment:
If `index.njk` extends `layout.njk`, will the degateway banner not already be included because it's in `layout.njk`?